website/static/js/filepage/index.js

```diff
@@ -226,40 +225,40 @@ var FileViewPage = {
                 m('.btn.btn-sm.btn-primary.file-download', {onclick: $(document).trigger.bind($(document), 'fileviewpage:download')}, 'Download')
             ]),
             m('.btn-group.btn-group-sm.m-t-xs', [
-                m('.btn.btn-default.disabled', 'Toggle view: ')
+               ctrl.editor ? m( '.btn.btn-default.disabled', 'Toggle view: ') : null
             ].concat(
                 m('.btn' + (ctrl.mfrIframeParent.is(':visible') ? '.btn-primary' : '.btn-default'), {
                     onclick: function (e) {
                         e.preventDefault();
-                        // atleast one button must remain enabled.
+                        // at least one button must remain enabled.
                         if (!ctrl.mfrIframeParent.is(':visible') || panelsShown > 1) {
                             ctrl.mfrIframeParent.toggle();
+                            ctrl.revisions.selected = false;
                         }
                     }
                 }, 'View')
-            ).concat(
-                panels.map(function(panel) {
-                    return m('.btn' + (panel.selected ? '.btn-primary' : '.btn-default'), {
-                        onclick: function(e) {
-                            e.preventDefault();
-                            // atleast one button must remain enabled.
-                            if (!panel.selected || panelsShown > 1) {
-                                panel.selected = !panel.selected;
-                            }
-                        }
-                    }, panel.title);
-                })
-            ))
+            ).concat([editPane()])
+            ),
+            m('.btn-group.m-t-xs', [
```

Comment:
For accessibility, clickable elements should be either `a` or `button` tags. This makes them tabbable and focusable (`divs` are not).